pages/pr/index.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from '@/app/page.module.css'
+
+export async function getStaticProps() {
```

Comment:
Remind me, why are we adding pages/boilerplaces for pages router into the app router branch? Do you think we should ask trainees to build all pages in both routers for comparison? Just making sure it's not misunderstood and "reflected" in the docs as well